pymodule.py

```diff
@@ -37,21 +37,29 @@
 def forte_driver(state_weights_list, scf_info, options, ints, mo_space_info):
 #    if options.get_str('PROCEDURE') == 'UNRELAXED':
 #        procedure_unrelaxed(...)
+    max_rdm_level = 3 # TODO: set this (Francesco)
 
     # Create an active space solver object and compute the energy
     active_space_solver_type = options.get_str('ACTIVE_SPACE_SOLVER')
     as_ints = forte.make_active_space_ints(mo_space_info, ints, "ACTIVE", ["RESTRICTED_DOCC"]);
     active_space_solver = forte.make_active_space_solver(active_space_solver_type,state_weights_list,scf_info,mo_space_info,as_ints,options)
+    active_space_solver.set_max_rdm_level(max_rdm_level)
     state_energies_list = active_space_solver.compute_energy()
 
-#    correlation_solver_type = options.get_str('CORRELATION_SOLVER')
-#    if correlation_solver_type != 'NONE':
+    correlation_solver_type = options.get_str('CORRELATION_SOLVER')
+    if correlation_solver_type != 'NONE':
+        reference = active_space_solver.reference()
+        semi = forte.SemiCanonical(mo_space_info, ints, options)
+        semi.semicanonicalize(reference, max_rdm_level)
+        Ua = semi.Ua_t()
```

Comment:
@fevangelista Do you have any idea what is wrong here (`semi.Ua_t()`)? It keeps saying `TypeError: Unable to convert function return value to a Python type! The signature was
	(self: forte.forte.SemiCanonical) -> ambit::Tensor` on my computer.

Reply:
Could the problem be that ambit::Tensor is not exported via pybind11?

Reply:
Ambit wrappers are available for Python, we should try to add `import ambit`

Reply:
Ok, it just need exposing `ambit::Tensor' via the one liner
```c++
    py::class_<ambit::Tensor>(m, "ambitTensor");
```

Reply:
Yes, thanks for the push!

Reply:
You're set to go. I just exposed `make_dynamic_correlation_solver`, but you will need to return an object before being able to use it.

Reply:
I currently expose the master_dsrg and the procedure should work for all unrelaxed dsrg methods.